CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ if(NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/src/cc/libbpf/src)
                   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
+include_directories(${KERNEL_INCLUDE_DIRS})
```

Comment:
who define this variable?

Reply:
I used it to be able to compile bcc with new kernel headers:

```
  $ cd <kernel-dir>
  $ make INSTALL_HDR_PATH=/tmp/headers headers_install
  $ cd <bpftrace-dir>
  $ cmake -DKERNEL_INCLUDE_DIRS=/tmp/headers/include/ ...
```

Reply:
I see. This could be useful in some cases. Could you add a comment in the cmake file and add more explanations in docs/reference_guide.md?

Reply:
sure, the comment is already in the changelog,
I'll add comments in cmake and ref guide then

Reply:
I did not find appropriate section in ref guide, I put comments into cmake file

Reply:
> I did not find appropriate section in ref guide, I put comments into cmake file

Okay. I guess in the future we should have a subsection in ref guide for CMAKE options.